src/Core/Makefile.am

```diff
@@ -7,3 +13,10 @@ SUBDIRS = \
 		FSpot.Query \
 		FSpot.Gui \
 		FSpot.Platform
+
+RESOURCES = \
+		FSpot.Core.UnitTest/TestData/taglib-sample.xmp \
+		FSpot.Core.UnitTest/TestData/taglib-sample-broken.xmp \
+		FSpot.Core.UnitTest/TestData/taglib-sample.jpg \
+		FSpot.Core.UnitTest/TestData/taglib-sample-broken.jpg
+		
```

Comment:
Sadly my Automake foo is pretty weak these days, but do we really want the distribution tarball to be able to run the unit tests? Or should the unit tests just be reserved for local development and testing?